cluster/src/main/java/org/apache/iotdb/cluster/server/Timer.java

```diff
@@ -183,14 +184,33 @@ public void add(long val) {
     }
 
     /**
-     * This method equals `add(System.nanoTime() - start)`. We wrap `System.nanoTime()` in this
-     * method to avoid unnecessary calls when instrumenting is disabled.
-     *
-     * @param start
+     * set now time as the operation's start time
      */
-    public void addNanoFromStart(long start) {
+    public void setStartTime() {
+      setStartTime(System.nanoTime());
+    }
+
+    public void setStartTime(long startTime) {
+      if (ENABLE_INSTRUMENTING) {
+        this.startTime = startTime;
+      }
+    }
```

Comment:
The condition should be put into the first method, as `System.nanoTime()` is a heavy call and should be avoided when not using instrumenting.

Reply:
Got it